events/envelope.proto

```diff
@@ -7,41 +12,53 @@ import "log.proto";
 import "metric.proto";
 import "error.proto";
 
-/// Envelope wraps an Event and adds metadata.
+// Envelope wraps an Event and adds metadata.
 message Envelope {
-    /// Type of the wrapped event.
+    // Type of the wrapped event.
     enum EventType {
-        // Removed Heartbeat at position 1
-        // Removed HttpStart at position 2
-        // Removed HttpStop at position 3
-        HttpStartStop = 4;
-        LogMessage = 5;
-        ValueMetric = 6;
-        CounterEvent = 7;
-        Error = 8;
-        ContainerMetric = 9;
+        reserved 1 to 3;
+        reserved "Heartbeat", "HttpStart", "HttpStop";
+        HTTP_START_STOP = 4;
```

Comment:
Read that changing field names can mess with JSON communication sometimes, so that probably isn't worth correcting. I should undo the field name changes to the messages and enums.